packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/task/TaskRequest.java

```diff
@@ -13,6 +13,7 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.models.ClientEventType;
```

Comment:
We cant do this, if we import from the hmiserver it will run the entire configuration.